modules/hal_nxp/mcux/mcux-sdk-ng/drivers/drivers.cmake

```diff
@@ -142,9 +142,14 @@ set_variable_ifdef(CONFIG_MIPI_DBI_NXP_DCNANO_LCDIF CONFIG_MCUX_COMPONENT_driver
 set_variable_ifdef(CONFIG_MIPI_DBI_NXP_FLEXIO_LCDIF CONFIG_MCUX_COMPONENT_driver.flexio_mculcd)
 set_variable_ifdef(CONFIG_VIDEO_MCUX_MIPI_CSI2RX    CONFIG_MCUX_COMPONENT_driver.mipi_csi2rx)
 set_variable_ifdef(CONFIG_ETH_NXP_IMX_NETC          CONFIG_MCUX_COMPONENT_driver.netc)
+
 set_variable_ifdef(CONFIG_ETH_NXP_IMX_NETC          CONFIG_MCUX_COMPONENT_driver.netc_switch)
 set_variable_ifdef(CONFIG_NXP_IRQSTEER1             CONFIG_MCUX_COMPONENT_driver.irqsteer_1)
 
+if(NOT CONFIG_SOC_MIMX9596)
+  set_variable_ifdef(CONFIG_ETH_NXP_IMX_NETC          CONFIG_MCUX_COMPONENT_driver.netc_switch)
+endif()
+
```

Comment:
Please fix this commit. Either drop/squash with the earlier change/fix the commit message.

Reply:
> Please fix this commit. Either drop/squash with the earlier change/fix the commit message.

fixed.